app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -50,6 +51,11 @@ BUILD_PREREQUIRES="
 
 defineDebugInfoPackage wordgrinder$secondaryArchSuffix \
 	$commandBinDir/wordgrinder
+PATCH()
+{
+	sed -i -e "s|/etc/dictionaries-common/words|$dataDir/wordgrinder-$portVersion/words/|*.dictionaries;" \
```

Comment:
```
	sed -i -e "s|/etc/dictionaries-common/words|$dataDir/wordgrinder-$portVersion/words/|*.dictionaries;" \
```
is incorrect for 2 reasons:
* using $portVersion here is not a good idea because $dataDir already has $portVersionedName in it.
* The `*.dictionaries` is not a sed command.